libyul/backends/evm/EVMDialect.cpp

```diff
@@ -137,6 +138,18 @@ set<YulString> createReservedIdentifiers(langutil::EVMVersion _evmVersion)
 	return reserved;
 }
 
+pair<YulString, BuiltinFunctionForEVM> createVerbatimWrapper(const std::string& _name, size_t _params, size_t _returns)
```

Comment:
Can we move this to another mixin-like file like zkEVMDialect.cpp?
Having as many changes as possible in separate files will make it easier eventually.

Reply:
I'm not sure about that at the moment. "createVerbatimWrapper" sounds like a general evm dialect API. We could move it to another file like EVMDialectUtils.cpp or something.

Also, I'm a little inclined towards introducing a zkevm dialect instead of modifying the evm dialect. I'm not sure about the cost and/or any other potential problem of doing it at the moment. But I think it will keep things clean.

Reply:
If we're not creating a new dialect, I still feel we can leave this function here. We already have changes in the createBuiltins() function in this file, so we'll anyways potentially run into merge conflicts when we rebase our development branch. Let me know if I'm missing any other motivation in moving this function to another file.